package.json

```diff
@@ -107,11 +110,12 @@
     "test:watch": "karma start --auto-watch=true --single-run=false",
     "test:ci": "npm-run-all test lint",
     "ciBuild": "npm-run-all sassBuild sassRender cssLint distJS bundler postinstall apiBuild",
-    "sassBuild": "npm-run-all sassBuild:demo sassBuild:component postCss:component sassRender",
-    "sassBuild:dev": "npm-run-all sassBuild:demo sassBuild:component postCss:component sassRender",
-    "sassBuild:demo": "node-sass ./demo/sass/ --output ./demo/css/",
-    "sassBuild:component": "node-sass ./src/ --output ./src/",
+    "sassBuild": "npm-run-all sassBuild:component postCss:component sassRender",
+    "sassBuild:dev": "npm-run-all sassBuild:component postCss:component sassRender",
+    "sassBuild:component": "sass --no-source-map src:src",
     "sassBuild:watch": "nodemon -e scss --watch src --exec npm run sassBuild:dev",
+    "sassBuild:debug": "sass —source-map scss/index.scss docs/index.css",
```

Comment:
Why was this added? I as configured I am not able to get this to work? 

``` npm ERR! code ELIFECYCLE
npm ERR! errno 64
npm ERR! @alaskaairux/auro-loader@1.1.1 sassBuild:debug: `sass —source-map scss/index.scss docs/index.css`
npm ERR! Exit status 64
```

Reply:
I followed the url to added the code, https://dev.to/ccreusat/migrating-from-node-sass-to-sass-dart-sass-with-npm-3g3c, i think it's my mistake sorry i will remove and fix as you suggest.